src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```
@@ -128,7 +128,8 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         virtual
         returns (uint mintAmount)
     {
-        _validateDepositAmount(_depositAmount);
```

One thing to ensure amount really never is 0 would be to check also in _calculateNetAndSplitFees that the total fees are not greater than or equal to BPS (it currently allows it to be equal which will result in amount 0)

---

When addressing the issue it became clear that changing this, i.e. making sure the check would be `totalFee < BPS` instead of `<=`, would only effect the read functions as we check that the output can't be 0 elsewhere already. 
1. We first check that `minAmountOut != 0`. [Link](https://github.com/InverterNetwork/contracts/blob/dev/src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol#L230)
2. We check that the actual amount out isn't lower than the `minAmountOut`. [Link](https://github.com/InverterNetwork/contracts/blob/dev/src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol#L282)
Same goes for the sell function

So in light of this we decided not to address this issue

---

Ok, I don't see any issue to leave that, but then I think it'll jjust be more accurate to make it < instead of <= since it is never meant to be equal and will be better to fail early.

---

Makes sense, @Zitzak addresses this [here](https://github.com/InverterNetwork/contracts/pull/619/commits/d4c37d1932543f6e36fc2c8609ace6a240355a03).